Core/src/main/java/com/plotsquared/core/plot/PlotArea.java

```diff
@@ -1452,6 +1452,20 @@ protected void setDefaultHome(BlockLoc defaultHome) {
         this.defaultHome = defaultHome;
     }
 
+    /**
+     * Get the maximum height that changes to plot components (wall filling, air, all etc.) may operate to
+     */
+    public int getMixComponentHeight() {
```

Comment:
```suggestion
    public int getMaxComponentHeight() {
```

is this method even used?

Reply:
Unused but added to match the new min method for parity